src/source/mod.rs

```diff
@@ -24,6 +25,7 @@ pub use self::file::{FileServer, FileServerConfig};
 pub use self::flush::{FlushTimer, FlushTimerConfig};
 pub use self::graphite::{Graphite, GraphiteConfig};
 pub use self::internal::{report_full_telemetry, Internal, InternalConfig};
+pub use self::nonblocking::{BufferedPayload, PayloadErr};
```

Comment:
Maybe `use self` rather than `pub use self`? Seems like these ought to be private to the source module, no?

Reply:
Certainly.  Fixed.